modules/Bio/EnsEMBL/Compara/PipeConfig/Plants/MergeDBsIntoRelease_conf.pm

```diff
@@ -60,6 +60,7 @@ sub default_options {
         'src_db_aliases' => {
             'master_db'     => 'compara_master',
             'protein_db'    => 'compara_ptrees',
+            'wheat_prot_db' => 'wheat_cultivars_ptrees',
```

Comment:
I think I would stick to naming this `wheat_ptrees`, but not important

Reply:
I have followed the same nomenclature as for vertebrates, but I don't mind either way.

Reply:
If I was reviewing those, I would say `pig_trees` and `mouse_trees` :grin: - it's :100: subjective and unimportant